PWGHF/D2H/TableProducer/candidateCreatorDV0Reduced.cxx

```diff
@@ -0,0 +1,254 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file candidateCreatorCharmResoReduced.cxx
```

Comment:
```suggestion
/// \file candidateCreatorDV0Reduced.cxx
```

Reply:
Actually it would be great to rename both the data creator and the candidate creator `CharmReso` since it is likely that in the future we will add channels also with charm baryons and not only D mesons (like SigmaCK0S), but it can also be done later

Reply:
Ok, I will change the filename then! Thanks to both of your comments!